CHANGELOG.rst

```diff
@@ -17,6 +17,15 @@ Documentation
 - Adds section on how to use arithmetic with ``PredictionEnsemble`` objects.
   (:pr:`378`) `Riley X. Brady`_.
 
+Internals/Minor Fixes
```

Comment:
Also add that `HindcastEnsemble.verify()` now accepts metric_kwargs

Reply:
I would also put this under Bugs for the stuff about `metric_kwargs` not working.